test/assets/multus/bridge-preexisting-pod.yaml

```diff
@@ -0,0 +1,28 @@
+kind: Pod
+apiVersion: v1
+metadata:
+  name: test-bridge-preexisting
+  annotations:
+    k8s.v1.cni.cncf.io/networks: bridge-preexisting-conf
+  labels:
+    app: test-bridge-preexisting
+spec:
+  terminationGracePeriodSeconds: 0
+  containers:
+  - name: hello-microshift
+    image: quay.io/microshift/busybox:1.36
+    command: ["/bin/sh"]
+    args: ["-c", "while true; do echo -ne \"HTTP/1.0 200 OK\r\nContent-Length: 16\r\n\r\nHello MicroShift\" | nc -l -p 8080 ; done"]
```

Comment:
Do we want to add "sleep" here not to burn CPU?

Reply:
This doesn't spin furiously - `nc` blocks until a client arrives, so the loop has a many iterations as many GETs.